module/zfs/sa.c

```diff
@@ -2237,6 +2277,7 @@ EXPORT_SYMBOL(sa_bulk_lookup);
 EXPORT_SYMBOL(sa_bulk_lookup_locked);
 EXPORT_SYMBOL(sa_bulk_update);
 EXPORT_SYMBOL(sa_size);
+EXPORT_SYMBOL(sa_size_locked);
```

Comment:
Is there a reason to export it? Or just for completeness?

(I don't think we really have any policy for what we export?)

Reply:
Not needed. Removed it.